scripts/scenarios/cloud7/qa/no-ssl/qa-scenario-2c.yaml

```diff
@@ -69,6 +69,7 @@ proposals:
 - barclamp: ceph
   attributes:
     disk_mode: first
+    client_network: admin
```

Comment:
@Itxaka I think the problem is pretty obvious, from chef/cookbooks/ceph/recipes/mon.rb :

```ruby

service "ceph_mon" do
  case service_type
  when "upstart"
    service_name "ceph-mon-all-starter"
    provider Chef::Provider::Service::Upstart
  when "systemd"
    service_name "ceph-mon@#{node["hostname"]}"
  else
    service_name "ceph"
  end
  supports restart: true, status: true
  action [:enable, :start]
  subscribes :restart, resources(template: "/etc/ceph/ceph.conf")
end
```

which starts the mon always on node["hostname"], aka the admin interface. what it should be started on is the public interface.